### PR TITLE
Include Project ID in GitLab summary

### PR DESCRIPTION
**Pull request by Alizter**

### CODE REVIEW

File: src/actions.ml [hunk 1]
```diff
@@ -444,26 +444,30 @@ let job_action ~bot_info ({build_name} as job_info) ~gitlab_mapping =
...
+    |> List.concat_map ~f:(fun stage ->
+           sorted_builds
+           |> List.filter_map ~f:(fun build ->
+                  if String.equal build.stage stage then
+                    Some
+                      (f "  - [%s](%s/-/jobs/%d)" build.build_name
+                         pipeline_info.common_info.repo_url build.build_id )
+                  else None )
+           |> List.cons ("- " ^ stage) )
+    |> String.concat ~sep:"\n\n"
```
Comment (Alizter) [inline src/actions.ml:464]: We can also decrease this to a single new line to stop the summary from being so spaced out.

File: src/actions.ml [hunk 2]
```diff
@@ -444,26 +444,30 @@ let job_action ~bot_info ({build_name} as job_info) ~gitlab_mapping =
...
+  in
+  [ summary_top
+  ; f "This [GitLab pipeline](%s) contains the following stages and jobs:"
+      pipeline_url
+  ; stage_summary
+  ; f "Project ID: %d" pipeline_info.common_info.project_id ]
```
Comment (Zimmi48) [inline src/actions.ml:470]: Maybe make this clearer like this:
```suggestion
  ; f "GitLab project ID: %d" pipeline_info.common_info.project_id ]
```

File: src/actions.ml (hunk 1 above)
Comment (Zimmi48) [reply src/actions.ml:464]: Sounds like a good idea.

File: src/actions.ml [hunk 3]
```diff
@@ -1623,11 +1627,8 @@ let coq_bug_minimizer_resume_ci_minimization_action ~bot_info ~key ~app_id body
       ; pr_number ] -> (
         message |> String.split ~on:'\n'
         |> function
-        | docker_image
-          :: target
-             :: opam_switch
-                :: failing_urls
-                   :: passing_urls :: base :: head :: bug_file_lines ->
+        | docker_image :: target :: opam_switch :: failing_urls :: passing_urls
+          :: base :: head :: bug_file_lines ->
```
Comment (Alizter) [inline src/actions.ml:1631]: OCaml format seems to have picked this up, shall I drop it?

Comment (Zimmi48) [reply src/actions.ml:1631]: Are you using 0.21? If yes, then let it be, and it's just me who needs to update.

**Review by Zimmi48 — APPROVED** [review]
I'll deploy the branch to test it (in progress at https://github.com/Zimmi48/bot/runs/5573186586?check_suite_focus=true).

**Pull request merged**